SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -20,13 +20,24 @@ function Invoke-lpass {
         $InputObject
     )
 
-    $lpassCommand = if ($null -ne $AdditionalParameters.lpassCommand){$AdditionalParameters.lpassCommand} else {'lpass'}
+    $lpassCommand = if ($null -ne $AdditionalParameters.lpassCommand){$AdditionalParameters.lpassCommand} else {''}
+    $lpassPath = if ($null -ne $AdditionalParameters.lpassPath){"`"$($AdditionalParameters.lpassPath)`""} else {'lpass'}
    
-    if ((Invoke-Expression -Command "$lpassCommand --version" ) -like 'LastPass CLI*') {
+    
+    if ($lpassCommand -ne '' -and ((& "$lpassCommand" "$lpassPath" --version ) -like 'LastPass CLI*') ) {
         if ($InputObject) {
-            return $InputObject | Invoke-Expression -Command "$lpassCommand @Arguments"
+            $InputObject | & "$lpassCommand" $lpassPath @Arguments
```

Comment:
I think lpassPath needs to be surrounded by quotes or we need to fix the example to say if you have a space in your path that you need to wrap it in quotes